src/ui/server/helpers/generate-full-application-by-reference-number-variables/index.test.ts

```diff
@@ -0,0 +1,39 @@
+import generateFullApplicationByReferenceNumberVariables from '.';
+import LOSS_PAYEE_ROUTES from '../../constants/routes/insurance/policy/loss-payee';
+import { INSURANCE_ROUTES } from '../../constants/routes/insurance';
+import { referenceNumber } from '../../test-mocks';
+
+const { LOSS_PAYEE_FINANCIAL_DETAILS_UK_ROOT } = LOSS_PAYEE_ROUTES;
+const { INSURANCE_ROOT, ALL_SECTIONS } = INSURANCE_ROUTES;
+
+describe('server/helpers/generate-full-application-by-reference-number-variables', () => {
```

Comment:
Could we rename this file/function to be 

```js
getApplicationByReferenceNumberVariables
```

So it's aligned with the GQL query?